test/unit/transaction/txs.js

```diff
@@ -1,544 +0,0 @@
-/* eslint-disable max-len */
-const transactions = {
```

Comment:
These were used only for transaction parsing testing (and the parsed format is now different in some aspects, eg. interop-lib returns `certificates: undefined` instead of `certificates: []`), so I think it doesn't make much sense to keep them.